bitbots_docs/docs/manual/tutorials/Docs-How-To.rst

```diff
@@ -2,10 +2,10 @@
 How to document
 ===============
 
-Our documentation is published under  `doku.bit-bots.de <http://doku.bit-bots.de>`_ and will automatically be regenerated from the package `bitbots_meta/bitbots_tools/bitbots_docs <https://github.com/bit-bots/bitbots_tools/tree/master/bitbots_docs>`_.
+Our documentation is published under `doku.bit-bots.de <https://doku.bit-bots.de>`_ and `docs.bit-bots.de <https://docs.bit-bots.de>`_ and will automatically be regenerated from the package `bitbots_meta/bitbots_tools/bitbots_docs <https://github.com/bit-bots/bitbots_tools/tree/master/bitbots_docs>`_.
```

Comment:
Do we need both URLs? Isn't the content the same?

Reply:
Yes, it's the same. But that way, people know both exist